README.md

```diff
@@ -0,0 +1,135 @@
+# mobile-android-authentication
+
+Implementation of Authentication package
```

Comment:
Is it worth linking the to the Authentication SDK confluence page here? https://govukverify.atlassian.net/wiki/spaces/DCMAW/pages/3816424843/Authentication+SDK+Specification

Reply:
Looking at this confluence, it is not 100% accurate to the actual implementation here. Also, iOS don't have that link yet, maybe something to consider and tidy up at a later stage...?

Reply:
What are the deviations? Are there good reasons for them?

Reply:
Currently;
- the `LoginSessionConfigruation` doesn't have the sealed classes
- there is no `cancel()` method on the `LoginSession` interface
- We don't have an error class (this we should and I will create a separate ticket to address this)

the first two points are in line with iOS, which also isn't quite matching the spec